sources/packages/backend/apps/workers/src/services/student-assessment/student-assessment.service.ts

```diff
@@ -176,6 +176,10 @@ export class StudentAssessmentService extends RecordDataModelService<StudentAsse
         { appealStatus: StudentAppealStatus.Approved },
       )
       .leftJoin("application.craIncomeVerifications", "craIncomeVerification")
+      .leftJoin(
+        "craIncomeVerification.supportingUser",
+        "incomeVerificationSupportingUser",
```

Comment:
Minor, since we have the `craIncomeVerification` for the student I would suggesting naming it as `craIncomeVerificationSupportingUser` to make the select looks like 
```
"craIncomeVerification.id",
"craIncomeVerification.craReportedIncome",
"craIncomeVerification.taxYear",
"craIncomeVerificationSupportingUser.id",
```